curve_fitting.ipynb

```diff
@@ -1,8 +1,18 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        """This function takes in one of the equations above, your x and y values, and then creates a graph of the fit data""""
</pre><p>docstring should start with one summary line, 60-80 characters, max. Can have longer text on subsequent lines (after a blank line).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/4-21-22-Class/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>